app/controllers/veterans_controller.rb

```diff
@@ -69,6 +83,21 @@ def set_veteran
 
     # Never trust parameters from the scary internet, only allow the white list through.
     def veteran_params
-      params.require(:veteran).permit(:first_name, :last_name, :email, :on_connect)
+      params.require(:veteran).permit(:first_name, 
+                                      :last_name, 
+                                      :email, 
+                                      :on_connect)
+    end
+
+    def veteran_connect_signup_params
+      params.require(:veteran).permit(:military_status, 
```

Comment:
since military_status is the same as roles, we can rollback that migration and just require "roles" here